cypress/e2e/cloud/annotations.test.ts

```diff
@@ -552,6 +558,197 @@ describe('The Annotations UI functionality', () => {
       cy.getByTestID('giraffe-annotation-tooltip').contains('im a hippopotamus')
     })
 
+    it('does not allow the form to be submitted if there is a validation error', () => {
```

Comment:
this is all client side validation, right? do we need an e2e test for this behavior? it seems more suited to a unit or integration test. you don't need to block this PR on moving this to an integration test, but I think we should move in that direction and try to avoid writing e2e tests if we can write other tests that give us confidence